gulp/html/copy.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+/**
+ * @function `gulp html:copy`
+ * @desc Copy the handlebars files to the assets folder.
+ */
+
+var gulp = require('gulp'),
+    htmlTask = require('./default');
+
+var src = ['source/**/*.hbs'],
```

Comment:
I extracted and set it in taskConfig (Line 13) because the pattern is exactly the same for src and watch.
Should I duplicate the pattern, or...?